### PR TITLE
Add Tutorials in language Spanish for LB4

### PR DESCRIPTION
**Pull request by HaroldV**

### CODE REVIEW

**Comment by HaroldV** [comment]
can you review my code ? @agnes512

File: _data/sidebars/es_lb4_sidebar.yml [hunk 1]
```diff
@@ -13,14 +13,14 @@ translated: true
 children:
 
 - title: 'Iniciando'
-  url: Getting-started.html
```
Comment (agnes512) [inline _data/sidebars/es_lb4_sidebar.yml]: Let's keep the url as `Getting-started.html`.

File: pages/es/lb4/iniciando.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
+---
+lang: es
+title: 'Iniciando'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar_es
+permalink: /doc/es/lb4/iniciando.html
```
Comment (agnes512) [inline pages/es/lb4/iniciando.md]: I remember that you had [this page](https://loopback.io/doc/es/lb4/Getting-started.html) translated last time. 
I think that one is good enough as we keep the **file names** and the **permalink** in English. 
This file can be removed. Thanks!

**Comment by agnes512** [comment]
Except those parts need to be modified, it looks good to me. 
Thank you for providing this translation!

File: pages/es/lb4/Tutorials.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+lang: es
+title: Tutorials
```
Comment (nabdelgadir) [inline pages/es/lb4/Tutorials.md:3]: Please add `layout: translation`

**Review by nabdelgadir — COMMENTED** [review]
+1 to @agnes512's comments

**Comment by HaroldV** [comment]
can you review newly please ? @agnes512 xD

**Review by agnes512 — APPROVED** [review]

**Review by nabdelgadir — APPROVED** [review]

**Comment by nabdelgadir** [comment]
Thanks again for the contribution!

**Pull request merged**